website/server/models/user/schema.js

```diff
@@ -9,6 +9,8 @@ import {
 
 const Schema = mongoose.Schema;
 
+const INVALID_DOMAINS = ['habitica.com', 'habitrpg.com'];
```

Comment:
Perhaps, if Habitica goes through another rebranding 😉
On Wed, Jun 15, 2016 at 10:49 AM Austin Witherow <notifications@github.com>
wrote:

> In website/server/models/user/schema.js
> <https://github.com/HabitRPG/habitrpg/pull/7659#discussion_r67188907>:
>
> > @@ -9,6 +9,8 @@ import {
> >
> >  const Schema = mongoose.Schema;
> >
> > +const INVALID_DOMAINS = ['habitica.com', 'habitrpg.com'];
>
> Do you think there will be more invalid domains eventually? If the list
> here grows, perhaps exporting stuff to a config file of sorts would be
> useful. Perhaps also over-engineering 😉
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7659/files/2161cc5748870bd5ae48a18fa32f4ac8522b3f95#r67188907>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALz_87GUteS_9ZB6o1eMTGZ2r2wghK6sks5qMB8EgaJpZM4I06jn>
> .
>